### PR TITLE
Add special error message for packet 20

### LINKED ISSUE #376: Improve error message about packet type 20
Add a short text that explains that we don't support LibrePGP's AEAD format, to help users to understand what's going on

### PR DESCRIPTION
**Pull request by DoumanAsh**
Fixes #376

### CODE REVIEW

File: src/packet/single.rs [hunk 1]
```diff
@@ -127,6 +127,7 @@ pub fn body_parser(ver: Version, tag: Tag, body: &[u8]) -> Result<Packet> {
         }
         Tag::ModDetectionCode => ModDetectionCode::from_slice(ver, body).map(Into::into),
         Tag::Padding => Padding::from_slice(ver, body).map(Into::into),
+        Tag::Other(20) => unimplemented_err!("GnuPG AEAD is unsupported and not part of OpenGPG"),
```
Comment (hko-s) [inline src/packet/single.rs:130]:
```suggestion
        Tag::Other(20) => unimplemented_err!("GnuPG-prorietary 'OCB Encrypted Data Packet' is unsupported"),
```

Comment (hko-s) [reply src/packet/single.rs:130]: Thanks for adding a message, this is useful! I suggested a minor change, above, to reflect the exact name of packet type 20 in the draft text of GnuPG's format.

Comment (hko-s) [reply src/packet/single.rs]: Oops, thanks for catching my typo :relaxed:

**Review by dignifiedquire — APPROVED** [review]
thanks, just one more `cargo fmt` is missing

**Pull request merged**